src/libraries/System.Private.CoreLib/src/System/Threading/PortableThreadPool.WorkerThread.cs

```diff
@@ -54,7 +59,7 @@ private static void WorkerThreadStart()
                         bool alreadyRemovedWorkingWorker = false;
                         while (TakeActiveRequest(threadPoolInstance))
                         {
-                            Volatile.Write(ref threadPoolInstance._separated.lastDequeueTime, Environment.TickCount);
+                            threadPoolInstance._separated.lastDequeueTime = Environment.TickCount;
```

Comment:
Is this prone to the recent TickCount resetting every 49 days issue?

Reply:
This value is used to take a diff from the current time later in time, though signed math is used and I have fixed the usage of it to use unsigned math. It's not a practical issue currently but at least it's less problematic to use unsigned math for spans of time.